pkg/services/playlist/playlistimpl/xorm_store.go

```diff
@@ -180,6 +181,51 @@ func (s *sqlStore) List(ctx context.Context, query *playlist.GetPlaylistsQuery)
 	return playlists, err
 }
 
+func (s *sqlStore) ListAll(ctx context.Context, orgId int64) ([]playlist.PlaylistDTO, error) {
+	db := s.db.GetSqlxSession() // OK because dates are numbers!
```

Comment:
What does your comment here mean, for my own edification?

Reply:
See the comment in https://github.com/grafana/grafana/blob/main/pkg/infra/db/db.go#L40

essentially in mysql, the magic time.Time conversions won't work unless the server has special startup flags.